docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,2005 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.        return discords_idx, discords_dist, discords_nn_idx
</pre><p>I am wondering if it makes sense to return lists or numpy arrays instead? I don't know</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>In fact, I thought about it before I pushed  the commits. The only thing that made me avoid doing that is function <code>_discords</code>  where we need to use <code>IDX</code> and <code>len(discords_idx)</code>. So, I thought, it might be hard to understand what's going on if we return numpy  ndarray <code>out</code>here in function <code>_find_discords</code>.</p><p>   </p><p>On the second thought, though, I think the structure of <code>out</code> in function <code>_discords</code> is similar to <code>mp.</code> So, it should not be hard for the reader once they understand what each column of <code>out</code> represents.      </p>

Reply:
<p>Btw, I think, if we return numpy ndarray, we can reduce the number of lines in function _discords without losing readability (given that reader know what each column represents). You can better see readability than me. Any opinion? </p>

Reply:
<p>I'll leave it to you to try and make a decision. Change it and see what you think. If it is worse then do not commit it.</p>

Reply:
Ok...so I will do some experiment on my end